apps/omg_watcher_info/lib/omg_watcher_info/db/transaction.ex

```diff
@@ -96,6 +96,39 @@ defmodule OMG.WatcherInfo.DB.Transaction do
     )
   end
 
+  @spec query_count :: Ecto.Query.t()
+  defp query_count do
+    from(transaction in __MODULE__, select: count())
+  end
+
+  @spec query_timestamp_between(Ecto.Query.t(), non_neg_integer(), non_neg_integer()) :: Ecto.Query.t()
+  def query_timestamp_between(query, start_datetime, end_datetime) do
+    from(transaction in query,
+      left_join: block in assoc(transaction, :block),
```

Comment:
```suggestion
      join: block in assoc(transaction, :block),
```

Reply:
Thanks Jacob for catching this and explaining. Done fdd87a1.